_ml-commons-plugin/ml-dashboard.md

```diff
@@ -6,8 +6,8 @@ redirect_from:
   - /ml-commons-plugin/ml-dashbaord/
 ---
 
-Released in OpenSearch 2.6, the machine learning (ML) functionality in OpenSearch Dashboards is experimental and can't be used in a production environment. For updates or to leave feedback, see the [OpenSearch Forum discussion](https://forum.opensearch.org/t/feedback-ml-commons-ml-model-health-dashboard-for-admins-experimental-release/12494).
-{: .warning }
+The ML dashboard was taken out of experimental status and released to General Availability in OpenSearch 2.9.  
```

Comment:
Do we want to include this note or do we just want to omit the experimental note? If we're including this, I think a note would be more appropriate than a warning.

Reply:
I'm okay changing it to a note.